templates/taxi/car_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% block content %}
+    <h1>Delete Car</h1>
+    <p>Aru you sure that you wont delete car {{ car }}?</p>
```

Comment:
There are a couple of typographical errors in this line. It should be: 'Are you sure that you want to delete car {{ car }}?'